.travis.yml

```diff
@@ -1,24 +1,16 @@
 ---
 env:
   global:
-    - DEFAULT_SQLITE_SQLALCHEMY__URL=sqlite:////tmp/tracim.sqlite
-    - DEFAULT_MYSQL_SQLALCHEMY__URL=mysql+pymysql://user:secret@127.0.0.1:3306/tracim_test
-    - DEFAULT_POSTGRES_SQLALCHEMY__URL=postgresql://user:secret@127.0.0.1:5432/tracim_test?client_encoding=utf8
     - DB=sqlite
     - COVERAGE=0
     - PSERVE_LOG=/tmp/pserve_log.log
-    - TRACIM_SQLALCHEMY__URL=$DEFAULT_SQLITE_SQLALCHEMY__URL
-    - TRACIM_DEPOT_STORAGE_NAME=test
-    - TRACIM_DEPOT_STORAGE_DIR=/tmp/test/depot
+    - TRACIM_UPLOADED_FILES__STORAGE__LOCAL__STORAGE_PATH=/tmp/test/depot
```

Comment:
This is possible, yes. For depot we could use the `memory` storage type.
In general it we could mount /tmp/ as a tmpfs. I'm not sure this is feasible within travis (but it could directly be done by their VM).

Reply:
Thanks for your answer.